src/Atmosphere.cpp

```diff
@@ -21,35 +20,20 @@ namespace csp::atmospheres {
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
-Atmosphere::Atmosphere(std::shared_ptr<Plugin::Properties> const& pProperties,
-    Plugin::Settings::Atmosphere const& settings, std::string const& sCenterName,
-    std::string const& sFrameName, double tStartExistence, double tEndExistence)
+Atmosphere::Atmosphere(std::shared_ptr<Plugin::Settings> const& settings,
+    std::string const& sCenterName, std::string const& sFrameName, double tStartExistence,
+    double tEndExistence)
     : cs::scene::CelestialObject(sCenterName, sFrameName, tStartExistence, tEndExistence)
-    , mRenderer(pProperties)
-    , mProperties(pProperties) {
+    , mRenderer(settings)
+    , mPluginSettings(settings) {
```

Comment:
Move by value and use `std::move` for all const references here.

Reply:
You're right but in order to avoid too many conflicts with the `feature/clang-tidy` I will do this after it got merged.